entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -299,6 +299,25 @@ public EMController(EntityRecordService entityRecordService,
 		// always return application/rdf+xml Content-Type
 	return createResponse(request, profile, type, languages, identifier, FormatTypes.xml,
 			HttpHeaders.CONTENT_TYPE_APPLICATION_RDF_XML);
+    }
+    
+	@ApiOperation(value = "Retrieve a known entity", nickname = "getEntitySchemaJsonLd", response = java.lang.Void.class)
+    @GetMapping(value = { "/{type}/base/{identifier}.schema.json", "/{type}/{identifier}.schema.json" },
```

Comment:
the extension for schemaOrg serialiazation is schema.jsonld like in entity API, please update the mapping.  http://entity-api-test.eanadev.org/docs/#/Entity_retrieval Ple

Reply:
Improved.